src/hyperlight_host/src/hypervisor/surrogate_process_manager.rs

```diff
@@ -160,7 +159,7 @@ impl SurrogateProcessManager {
                 mapping_file_handle,
                 surrogate_process_handle,
                 0,
-                Some(raw_source_address),
+                None,
```

Comment:
Probably change raw_size below here to 0

Reply:
Probably not thinking more about it... We use it to set guard pages, so we should probably use it here explicitly anyway.